src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -49,7 +50,8 @@ def test_basic_factory_logic(
         assert ContainerSerializer.Meta.embedded_fields == ["cluster"]
         assert isinstance(ContainerSerializer.cluster, EmbeddedField)
         assert ContainerSerializer.cluster.related_model is afval_cluster_model
-        assert ContainerSerializer.cluster.serializer_class is ClusterSerializer
+        # Comparing classes in here, because EmbeddedField.serializer_class is not created yet.
+        assert ContainerSerializer.cluster.serializer_class.__class__ is ClusterSerializer.__class__
```

Comment:
Je vergelijkt nu hier de metaclasses met elkaar, dat is altijd true.

Het doel was juist om te controleren dat de relaties dezelfde class gebruikten; dat er geen onnodige dubbele serializer classes gegenereerd werden.

Reply:
Veranderd check en toegevoegd extra omschrijving.